app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -91,3 +91,8 @@ input.search-bar {
 .hi-xl {
   width: 72px;
 }
+
+// Eventually remove once spacing in className works for avatar in AdminUserRow
+.avatar {
+  margin-right: 5px;
+}
```

Comment:
You can add `<div className="me-2">` to avatar parent div instead of having an extra CSS class.